web/docs/parameters.md

```diff
@@ -0,0 +1,18 @@
+---
+sidebar_position: 100
+---
+
+# Parameters
+
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+
+The following parameters are available for the Barman Cloud Plugin:
+
+- `barmanObjectName`: references the `ObjectStore` resource to be used by the
+  plugin.
+- `serverName`: Specifies the server name in the object store.
+
+:::important
```

Comment:
Mhhh ... Why is that and why don't we mention this in the API?

Reply:
We should also at least mention this issue here: https://cloudnative-pg.io/plugin-barman-cloud/docs/next/usage/#defining-the-objectstore

Reply:
The api documentation for `serverName` in the barman configuration is generated from `barman-cloud` module, so it is also included in the in-tree implementation in CNPG.

We could include the notice in the global documentation of the ObjectStore type, together with a CIL rule to prevent the value from being set.

Reply:
Created #334 to fix this

Reply:
@gbartolini, please check now. I've added an admonition in the usage page and changed the message according to #334

Reply:
awesome, thanks.